app/resources/errors.html.template

```diff
@@ -8,6 +8,13 @@
       class="mandatory" style="display: none">
     {% trans "Please fill in all the required fields." %}
   </div>
+  <div id="email_id_improper_format"
+      class="mandatory" style="display: none">
+  {% blocktrans %}
+  The Email address you entered doesn't look like valid address.
```

Comment:
To shorten it and make localization easier.

"The Email address you entered is not valid."